rails/app/models/concerns/importable.rb

```diff
@@ -120,7 +120,7 @@ def save_importable_records
 
         # Find or create has_many* relationships
         @klass.associated_attribute_names.each do |association|
-          values = attributes[association].to_s.split(",")
+          values = attributes[association].to_s.split(",").map(&:strip)
```

Comment:
Rather than mapping here, I think we can do it when we loop on line 125 `create__by(name: name.strip, …)`. It will save us from doing the same map loop twice and it will set a precedent for any other data manipulation we might need to do in the future.